AE_PunktBeschriftung/CMakeLists.txt

```diff
@@ -6,10 +6,10 @@ set(CMAKE_CXX_STANDARD 20)
 add_executable(label_map src/main/main.cpp
         src/main/generator/InstanceGenerator.cpp
         src/main/representations/Instance.cpp
-        src/main/representations/Solution.cpp
+        src/main/representations/SolutionWithGraph.cpp
         src/main/representations/Point.cpp
         src/main/solver/TrivialSolver.cpp 
-        src/main/io/InstanceReader.cpp)
+        src/main/io/InstanceReader.cpp src/main/representations/Solution.cpp include/representations/Solution.hpp)
```

Comment:
I'm not sure that adding the .hpp file is necessary here

Reply:
You're right. My IDE does it automatically -.-